app/views/tournaments/_deck_selection.html.slim

```diff
@@ -0,0 +1,25 @@
+.col-md-6
+  .card
+    .card-header
+      | Please select from your decks below.
+      a< href="https://netrunnerdb.com/en/decks" target="_blank" Set them in NetrunnerDB.
+    ul.list-group.list-group-flush#nrdb_decks_selected style='border-bottom: 0'
+      li.list-group-item.selected-deck
+        .selected-deck-identity style="background-image:url(https://static.nrdbassets.com/v1/small/00006.jpg)"
```

Comment:
It surprised me that clicking this cleared out my deck selection.

How about a button to clear it?

Reply:
I've done that. Good spot, thanks.